### DISCUSSION TITLE
The "Ship of fools" allegory originated in a philosophical text by who?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#519] [Galaxy Trivia] [answered]
⛵🤪🤡

**Comment by NobleOsinachi** [comment]
The "Ship of Fools" allegory is often attributed to the German humanist and satirist Sebastian Brant, who wrote a book titled "Das Narrenschiff" (The Ship of Fools) in 1494.

**Comment by thuongtruong1009** [comment, marked as answer]
The "Ship of Fools" allegory originated in a satirical allegorical work called "Das Narrenschiff" (The Ship of Fools), written by the German humanist and theologian Sebastian Brant in 1494. The work was a critique of the moral and intellectual decay of society at the time, and it used the metaphor of a ship carrying fools to illustrate the foolishness and vices of humanity. The work became very popular and was widely read throughout Europe in the 16th century, and the allegory of the ship of fools has since become a common motif in art and literature.

**Comment by ParanoidUser** [comment]
Sebastian Brant made this allegory shine in his book. But it originated long before that, around 375 BC. Can you guess the author? 🏛️🏺

**Comment by ErKiran** [comment]
The "Ship of fools" allegory originated in a philosophical text by Plato.

**Reply by ParanoidUser** [reply]
Bingo! Thank you, Kiran! 👏
